### ISSUE TITLE
RTL Support

### ISSUE THREAD

**Issue by mohamadian** [mxck/react-native-material-menu#69]
The menu is not shown when I use an RTL user interface. The position of the menu button is incorrect. In function **show()** at line 89, the **this._container.measureInWindow()** function return wrong "Left" position of the menu button.
To solve the problem, in **render()** function at line 146, I just add the following code. It works for me.

```
if (I18nManager.isRTL == true)
      left = windowWidth-buttonWidth-left;
```

The **render()** function looks like this:

```
render() {
    const dimensions = Dimensions.get('window');
    const { width: windowWidth } = dimensions;
    const windowHeight = dimensions.height - (StatusBar.currentHeight || 0);

    const {
      menuSizeAnimation,
      menuWidth,
      menuHeight,
      buttonWidth,
      buttonHeight,
      opacityAnimation,
    } = this.state;
    const menuSize = {
      width: menuSizeAnimation.x,
      height: menuSizeAnimation.y,
    };
    
    // Adjust position of menu
    let { left, top } = this.state;
    if (I18nManager.isRTL == true)
      left = windowWidth-buttonWidth-left;    
    const transforms = [];

    .
    .
    .
    
  }

```

**Comment by mohamadian** [comment]
The above solution is not working in some case. It needs your help.

**Comment by yaron1m** [comment]
I added a pull request #73

**Comment by mxck** [comment]
Closed in 35ad4c034ea39ee06908e46b15cc55fcbb440f51
